appinventor/codelabs/talk-to-me/index.html

```diff
@@ -0,0 +1,163 @@
+
+<!doctype html>
+
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <meta name="theme-color" content="#4F7DC9">
+  <meta charset="UTF-8">
+  <title>Talk To Me Codelab</title>
+  <script src="../../bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
+  <link rel="import" href="../../elements/codelab.html">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Code+Pro:400|Roboto:400,300,400italic,500,700|Roboto+Mono">
+  <link rel="icon" href="../../elements/favicon.png">
+  <style>
+    body {
+      font-family: "Roboto",sans-serif;
+    }
+  </style>
+
+</head>
+<body unresolved class="fullbleed">
+
+  <google-codelab title="Talk To Me Codelab" feedback-link="https://goo.gl/forms/9zDi1V3arg167lIY2">
+
+      <google-codelab-step label="Overview">
+        <p>In this codelab, you&#39;ll learn how to make your first app!
+        <h3 class="checklist">What you&#39;ll learn</h3>
+        <ul class="checklist">
+          <li>Use the <b>Designer View</b> to create the user interface.</li>
+          <li>Use the <b>Blocks View</b> to add functionality to the app.</li>
+          <li>Use the <b>Button</b> and <b>TextToSpeech</b> components.</li>
+          <li>Use <b>Event Handlers</b>.</li>
+        </ul>
+
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Create a new project">
+        <p>Create a <a href="http://ai2.appinventor.mit.edu/" target="_blank">new project</a> called
+          <code>TalkToMe</code> (no spaces).</p>
+        <img src="img/new_project.png">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add a Button">
+        <p>The <b>Designer View</b> shows you what the app looks like to the user.</p>
+        <p>In the <b>Palette</b> section, under the <b>User Interface</b> drawer and drag a <b>Button</b> component onto the viewer.</p>
+        <img src="img/button.png">
+        <p>Next, in the <b>Media</b> drawer, drag a <b>TextToSpeech</b> component onto the viewer.</p>
+        <p>Notice, it appears in the <b>Non-visible components</b> section under the viewer.</p>
+        <h2>Challenges</h2>
+        <ol>
+          <li>Change the text of the Button to say "Talk To Me". <i>Hint: Use the Properties section of the Button component.</i></li>
```

Comment:
Can you add a screenshot of the properties screen? This is the first time we're mentioning it.